src/SprykerEco/Glue/UnzerRestApi/Plugin/CheckoutRestApi/UnzerCheckoutDataResponseMapperPlugin.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Plugin\CheckoutRestApi;
+
+use Generated\Shared\Transfer\RestCheckoutDataResponseAttributesTransfer;
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+use Generated\Shared\Transfer\RestCheckoutRequestAttributesTransfer;
+use Spryker\Glue\CheckoutRestApiExtension\Dependency\Plugin\CheckoutDataResponseMapperPluginInterface;
+use Spryker\Glue\Kernel\AbstractPlugin;
+
+/**
+ * @method \SprykerEco\Glue\UnzerRestApi\UnzerRestApiFactory getFactory()
+ */
+class UnzerCheckoutDataResponseMapperPlugin extends AbstractPlugin implements CheckoutDataResponseMapperPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Maps `RestCheckoutDataTransfer.unzerCredentials.unzerKeypair.publicKey` to `RestCheckoutDataResponseAttributesTransfer.unzerPublicKey`.
+     *
```

Comment:
Please add required fields which should be set for method parameters

Reply:
Fixed.